components/eamxx/tests/single-process/mam/aero_microphys/input.yaml

```diff
@@ -11,21 +11,21 @@ time_stepping:
 atmosphere_processes:
   atm_procs_list: [mam4_aero_microphys]
   mam4_aero_microphys:
-   mam4_linoz_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/linoz1850-2015_2010JPL_CMIP6_10deg_58km_ne2np4_c20240724.nc
-   mam4_oxid_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/oxid_1.9x2.5_L26_1850-2015_ne2np4L72_c20240722_OD.nc
-   mam4_linoz_chlorine_file : ${SCREAM_DATA_DIR}/mam4xx/physprops/Linoz_Chlorine_Loading_CMIP6_0003-2017_c20171114.nc
-   mam4_chlorine_loading_ymd : 20100101
-   mam4_rsf_file : ${SCREAM_DATA_DIR}/mam4xx/physprops/RSF_GT200nm_v3.0_c080811.nc
-   mam4_xs_long_file : ${SCREAM_DATA_DIR}/mam4xx/physprops/temp_prs_GT200nm_JPL10_c130206.nc
-   mam4_so2_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_so2_elev_ne2np4_2010_clim_c20240726_OD.nc
-   mam4_so4_a1_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_so4_a1_elev_ne2np4_2010_clim_c20190821_OD.nc
-   mam4_so4_a2_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_so4_a2_elev_ne2np4_2010_clim_c20190821_OD.nc
-   mam4_pom_a4_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_pom_a4_elev_ne2np4_2010_clim_c20190821_OD.nc
-   mam4_bc_a4_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_bc_a4_elev_ne2np4_2010_clim_c20240726_OD.nc
-   mam4_num_a1_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_num_a1_elev_ne2np4_2010_clim_c20190821_OD.nc
-   mam4_num_a2_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_num_a2_elev_ne2np4_2010_clim_c20190821_OD.nc
-   mam4_num_a4_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_num_a4_elev_ne2np4_2010_clim_c20190821_OD.nc
-   mam4_soag_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_soag_elev_ne2np4_2010_clim_c20190821_OD.nc
+   mam4_linoz_file_name : ${SCREAM_DATA_DIR}/mam4xx/linoz/ne2np4/linoz1850-2015_2010JPL_CMIP6_10deg_58km_ne2np4_c20240724.nc
+   mam4_oxid_file_name : ${SCREAM_DATA_DIR}/mam4xx/invariants/ne2np4/oxid_1.9x2.5_L26_1850-2015_ne2np4L72_c20240722_OD.nc
+   mam4_linoz_chlorine_file : ${SCREAM_DATA_DIR}/mam4xx/linoz/Linoz_Chlorine_Loading_CMIP6_0003-2017_c20171114.nc
+   mam4_chlorine_loading_ymd : 20100101   
+   mam4_rsf_file : ${SCREAM_DATA_DIR}/mam4xx/photolysis/RSF_GT200nm_v3.0_c080811.nc
+   mam4_xs_long_file : ${SCREAM_DATA_DIR}/mam4xx/photolysis/temp_prs_GT200nm_JPL10_c130206.nc
+   mam4_so2_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/emissions/ne2np4/elevated/cmip6_mam4_so2_elev_ne2np4_2010_clim_c20240726.nc    
```

Comment:
These files also need to be added to the namelist file.

Reply:
Yes, I will do that in the next iteration.